### DISCUSSION TITLE
Starred repo doesn't appear on my stars page

### DISCUSSION THREAD

**Discussion by energister** [community/community#29904] [Other Feature Feedback, Questions, & Ideas]
One of the internal repos in my organization which I have starred doesn't appear in my stars list.
So on the repo page I see my star but there is no such repo on https://github.com/energister?tab=stars nor on the https://github.com/stars/energister/repositories?filter=all page. I've already tried to toggle star on and off without success.

Strangely enough, but starring another internal repo in the same organization works perfectly fine: it appears and disappears from the mentioned lists immediately.

The difference I've noticed between these two repos is that the first one (that doesn't behave properly) has no stars except mine.

**Comment by NikhilMath** [comment]
I have seen this problem a lot, from what I gather, it is a glitch and GitHub will eventually fix it, but yeah, its not supposed to be like that.

**Comment by energister** [comment]
5 days left - no changes. The issue is still reproducing

**Comment by v1ctorio** [comment]
is the repository private?

**Comment by energister** [comment]
The other strange thing is that the Internal repo that IS in the starred list is marked with the "Private" label there (whereas on the repo page it has Internal label)

@victorioxd No, the problem repo is Internal

**Comment by iamtodor** [comment]
I have the same issues within the organization

**Comment by nagendra-prabhu** [comment]
I too had the same issue in my organization.
But later I found that it is showing all stared repo after I click single sign on link that pops up.
